### ISSUE TITLE
VSCode 1.84 - Status bar position

### ISSUE THREAD

**Issue by TauerOne** [drcika/apc-extension#99]
I just updated to VSCode 1.84.  My statusbar position setting of "editor-top" is no longer having any effect.  Setting it to "top" works, although part of the statubar is now behind the minimize/restore/close (stoplight) buttons on Windows.  Setting it to "bottom" and "editor-bottom" also work.

**Comment by florius0** [comment]
It seems that the problem is in 1.84. On my setup it feels as APC just stopped working. No custom css is being applied, electron settings are also ignored

**Comment by florius0** [comment]
Solution: https://github.com/drcika/apc-extension/issues/90

**Comment by electrovir** [comment]
This appears to be even worse in 1.85. Status bar position now has no effect at all. No matter what value I set it to, nothing changes.

**Comment by rgsx** [comment]
VS Code just had an update and some APC options are broken. Status Bar options are ignored.

> Solution: #90

Unfortunately, this solution no longer works.

**Comment by Xelbayria** [comment]
if you create a custom profile aside from "Default", then it should be working without any problem. I was able to get this working on v1.84. I couldn't find one setting in Profile "Default" that allowed the change from **APC Customize UI++** to be in effect. 

hope this help. It looks like I may have to put v1.85 on hold for now. 

nvm. turned out it was my workspace profile that overrides the settings.json. I was able to get it working in "Default". 

EDIT: 
I made a mistake. This is not related to _Status Bar_, this is related to _Activity Bar_. Just tested _Status Bar_ on V1.84 and it's working fine.
